client/overview/task-list/tasks/po-task.tsx

```diff
@@ -34,6 +34,7 @@ export const getVerifyBankAccountTask = (): any => {
 
 		window.location.href = addQueryArgs( wcpaySettings.connectUrl, {
 			collect_payout_requirements: true,
+			source: 'overview-page__receive-deposits-task',
```

Comment:
Maybe we should capture that this is a po task like `overview-page__po-verify-bank-account-task`

Reply:
Addressed in https://github.com/Automattic/woocommerce-payments/pull/8668/commits/6805615ab089eb023e57737d80ae7d96248ed2c4